README.md

```diff
@@ -1,7 +1,5 @@
 # AlternateLife.RageMP.Net
-![Windows build status](https://teamcity.alternate-life.de/app/rest/builds/buildType:(id:RageMP_NETCore_Release)/statusIcon) (Windows)
-![Linux build status](https://teamcity.alternate-life.de/app/rest/builds/buildType:(id:RageMP_NETCore_Linux_Release)/statusIcon) (Linux)
-[![Docker build status](https://teamcity.alternate-life.de/app/rest/builds/buildType:(id:RageMP_NETCore_Linux_Docker_Release)/statusIcon)](https://cloud.docker.com/repository/docker/alternatelife/ragemp-server) (Docker)
+[![Development Build status](https://teamcity.alternate-life.de/app/rest/builds/buildType:(id:RageMP_NETCore_BuildDevelopment)/statusIcon)](https://teamcity.alternate-life.de/viewType.html?buildTypeId=RageMP_NETCore_BuildDevelopment)
```

Comment:
I would like to add `(Windows, Linux)` to the build status to make clear it covers both